client/my-sites/upgrades/domain-management/transfer/utils.js

```diff
@@ -0,0 +1,101 @@
+/**
```

Comment:
The name of the file is kinda confusing - we have two functions that display notices/errors and one related to promises. Only the last one seems to me at home in a file named `utils.js`. Maybe the other two could be moved to `common-notices.js` or something like that?